examples/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -149,8 +149,10 @@
 device = 'cuda' if cuda else 'cpu'
 if cuda:
     torch.backends.cudnn.benchmark = True
-seed = 20200220  # random seed to make results reproducible
-# Set random seed to be able to reproduce results
+# Set random seed to be able to roughly reproduce results
+# Note that with cudnn benchmark set to True, GPU indeterminism
+# may still make results substantially different between runs
```

Comment:
I would clearly explain that this can be done by setting cudnn_benchmark=True in set_random_seeds